editor/components/block-mover/index.js

```diff
@@ -45,6 +45,7 @@ export class BlockMover extends Component {
 	render() {
 		const { onMoveUp, onMoveDown, isFirst, isLast, uids, blockType, firstIndex, isLocked, instanceId, isHidden } = this.props;
 		const { isFocused } = this.state;
+		const countBlocks = castArray( uids ).length;
```

Comment:
Minor nitpick: countBlocks sounds like a function name to me as count is a verb, maybe we can name it numberOfBlocks or even blocksCount.